flow/project.py

```diff
@@ -571,11 +718,15 @@ def __init__(self, cmd, pre=None, post=None):
     def __str__(self):
         return "{type}(cmd='{cmd}')".format(type=type(self).__name__, cmd=self._cmd)
 
-    def __call__(self, job=None):
+    def __call__(self, jobs=None):
         if callable(self._cmd):
-            return self._cmd(job).format(job=job)
+            return self._cmd(*jobs).format(jobs=' '.join(map(str, jobs)))
```

Comment:
```suggestion
            return self._cmd(*jobs).format(*jobs))
```
This is likely what is needed here. The string with spaces may not be the format needed by the end user, so we should not force that on him.